### PR TITLE
Use Variables, update dependencies

### PR DESCRIPTION
**Pull request by phillipivan**
- [FEAT] New Action Set Value Integer from Variable. Addresses #27 
- [FEAT] New Feedback Parameter Equals String
- [FEAT] Accept negative values for Feedback Parameter Equals
- [FEAT] Accept variable for action option pathInput
- [FEAT] Accept variable for action Set Value String
- More status updates from setupEmberConnection
- [CHORE] Update engine to node 18.20
- [CHORE] Specify package manager Yarn 4.4.1
- [CHORE] Update companion-module-base to 1.10.0
- [CHORE] Update companion-module-tools to 2.0.3
- [CHORE] Update @types/node to 18.19.45
- [CHORE] Update husky to 9.1.6
- [CHORE] Update lint-staged to 15.2.0
- [CHORE] Include prettier at 3.3.3
- [CHORE] Include eslint at 9.10.0
- [CHORE] Update rimraf to 5.0.10
- [CHORE] Update typescript to 5.5.4
- [CHORE] Update typscript-eslint to 8.5.0
- [CHORE] Update scripts
- [CHORE] Version bump to 2.3.0

### CODE REVIEW

File: .yarn/install-state.gz
Comment (Julusian) [inline .yarn/install-state.gz:1]: This shouldn't be committed

**Review by Julusian — APPROVED** [review]

File: .yarn/install-state.gz
Comment (phillipivan) [reply .yarn/install-state.gz:1]: Ok. Should that be part of the gitignore when using yarn 4?

Comment (Julusian) [reply .yarn/install-state.gz:1]: yes it should. I think I have started ignoring the whole `.yarn` folder

Comment (phillipivan) [reply .yarn/install-state.gz:1]: I copied what you have in the atem module... but I think I did this after that would have been in the repo already.

Comment (phillipivan) [reply .yarn/install-state.gz:1]: > .yarn/*
> !.yarn/patches
> !.yarn/plugins
> !.yarn/releases
> !.yarn/sdks
> !.yarn/versions
>

Comment (Julusian) [reply .yarn/install-state.gz:1]: oh yeah, at least one of those folders I have used (patches), but others (versions) seem to get empty files added sometimes that need ignoring. 
Either way is fine with me

**Pull request merged**